ccd-definition/FixedLists/CareSupervision/DirectionAssignee.json

```diff
@@ -1,44 +0,0 @@
-[
```

Comment:
in `ccd-definition/FixedLists/CareSupervision/gatekeepingOrder/directionAssignee.json` the display ordering of the last 2 fields might lead to inconsistent ouput

Reply:
good spot, done